test/e2e/util/util.go

```diff
@@ -144,11 +144,7 @@ func WaitForNReadyNodes(t *testing.T, ctx context.Context, client crclient.Clien
 	start := time.Now()
 
 	// waitTimeout for nodes to become Ready
-	waitTimeout := 30 * time.Minute
-	switch platform {
-	case hyperv1.PowerVSPlatform:
-		waitTimeout = 60 * time.Minute
-	}
+	waitTimeout := 60 * time.Minute
```

Comment:
what is causing this increase?

Reply:
it takes longer for nodes to get ready with the mitm proxy
```
=== CONT  TestCreateClusterProxy
    util.go:190: All nodes are ready. Count: 2
    util.go:196: All nodes for nodepool appear to be ready in 53m50s. Count: 2
```